camp/templates/worldmap/geo_header.html

```diff
@@ -0,0 +1,25 @@
+{% load api_basemaps_tags %}
+{% load i18n %}
+{% load static from staticfiles %}
+
+<link rel="stylesheet" type="text/css" href="{% static "worldmap_client/theme/site.css" %}?v={{ VERSION }}" />
+<link rel="stylesheet" type="text/css" href="{% static "worldmap_client/externals/ext/resources/css/ext-all.css" %}?v={{ VERSION }}" />
+<link rel="stylesheet" type="text/css" href="{% static "worldmap_client/externals/openlayers/theme/default/style.css" %}?v={{ VERSION }}" />
+<link rel="stylesheet" type="text/css" href="{% static "worldmap_client/externals/geoext/resources/css/geoext-all.css" %}?v={{ VERSION }}" />
+<link rel="stylesheet" type="text/css" href="{% static "worldmap_client/externals/gxp/theme/all.css" %}?v={{ VERSION }}">
+<link rel="stylesheet" type="text/css" href="{% static "worldmap_client/externals/ext/resources/css/xtheme-white.css" %}?v={{ VERSION }}" />
+<link rel="stylesheet" type="text/css" href="{% static "worldmap_client/theme/app/geoexplorer_white.css" %}?v={{ VERSION }}" />
+<link rel="stylesheet" type="text/css" href="{% static "worldmap_client/theme/ux/colorpicker/color-picker.ux.css" %}?v={{ VERSION }}" />
+<link rel="stylesheet" type="text/css" href="{% static "worldmap_client/theme/ux/spinner/Spinner.css" %}?v={{ VERSION }}" />
+
+<script src="{% static "worldmap_client/script/OpenLayers.js" %}?v={{ VERSION }}" type="text/javascript"></script>
+<script src="{% static "worldmap_client/script/GeoExt.js" %}?v={{ VERSION }}" type="text/javascript"></script>
+<script src="{% static "worldmap_client/script/gxp.js" %}?v={{ VERSION }}" type="text/javascript"></script>
+<script src="{% static "worldmap_client/script/GeoExplorer.js" %}?v={{ VERSION }}" type="text/javascript"></script>
+<script src="{% static "worldmap_client/script/GeoNode.js" %}?v={{ VERSION }}"></script>
+
```

Comment:
this would be better to be handled directly in GeoNode with a {{ if GOOGLE_API_KEY }} block.
I will send the PR to GeoNode right today, so please remove this file from the PR.